tests/CoordinateControllerTest.cs

```diff
@@ -97,63 +188,116 @@ private async Task AssertMigrateCoordinatesAsync(bool onlyMissing, int transform
             .Verify("SendAsync", Times.Exactly(transformedCoordinatesCount), ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>());
     }
 
-    private void AssertBohrungJacquelyn30()
+    private Borehole CreateLV95BoreholeWithAllCoordinatesSet()
     {
-        var bohrung = context.Boreholes.Single(b => b.Id == 1000006);
-        Assert.AreEqual("Laurence.Padberg3", bohrung.AlternateName);
-        Assert.AreEqual(ReferenceSystem.LV95, bohrung.OriginalReferenceSystem);
-        Assert.AreEqual(2626103.1988343936, bohrung.LocationX);
-        Assert.AreEqual(1125366.3469565178, bohrung.LocationY);
-        Assert.AreEqual("POINT (2626103.1988343936 1125366.3469565178)", bohrung.Geometry.ToString());
-        Assert.AreEqual(9876.5435435435, bohrung.LocationXLV03);
-        Assert.AreEqual(1234.5623562356, bohrung.LocationYLV03);
+        return context.AddNewBoreholeToContext(new Borehole()
```

Comment:
PP: Bei Objekt-Initialisierungen mit Werten können die Klammern weggelassen werden:
```suggestion
        return context.AddNewBoreholeToContext(new Borehole
```

Gilt auch für alle anderen Fälle weiter unten.